### DISCUSSION TITLE
Library apa aja yang kepake?

### DISCUSSION THREAD

**Discussion by Zzathy** [nauzyx-labs/pintarin#2] [Q&A] [answered]
Library apa aja yang kepake?

**Comment by xyzuan** [comment, marked as answer]
Library yg penting _Firebase, Google Material, ExoPlayer2 dan Retrofit_

**Reply by xyzuan** [reply]
Bisa dilihat di app gradle sebagai referensi library yang dipakai https://github.com/xyz-prjkt/pintarin/blob/master/app/build.gradle
